modules/nf-core/coptr/extract/main.nf

```diff
@@ -0,0 +1,49 @@
+process COPTR_EXTRACT {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/coptr:1.1.4--pyhdfd78af_3':
+        'biocontainers/coptr:1.1.4--pyhdfd78af_3' }"
+
+    input:
+    tuple val(meta), path(bam, stageAs: "bamfolder/*")
```

Comment:
Why are these being staged into the bamfolder, rather than just into the directory?
Can the tool not take `.` as the folder containing the bams?

Reply:
Not really, the tool requires a folder, it's a bit strict that way.